src/components/quiz/QuestionContent.tsx

```
@@ -0,0 +1,16 @@
+import { renderContent } from '@/lib/utils/render-content';
+
+interface QuestionContentProps {
+  content: any; // Use a more specific type if available
+}
+
+export default function QuestionContent({ content }: QuestionContentProps) {
+  return (
+    <div
+      className="prose max-w-none"
+      dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)